### PR TITLE
feat: romanian

### PR DESCRIPTION
**Pull request by MrlolDev**
Added full romanian translation, I am not native speaker so it can has some errors of expressions but the same happens with other languages.

### CODE REVIEW

**Comment by MrlolDev** [comment]
Any help with the error pls?

**Comment by AbdBarho** [comment]
@MrlolDev rebase your code on top of main, the error should be fixed.

File: website/public/locales/ro/side_menu.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "dashboard": "Panou de control",
```
Comment (fozziethebeat) [inline website/public/locales/ro/side_menu.json:2]: I believe this file was removed.  Can you move these entries into `common.json`?

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
If possible, could up merge in the recent changes from main? That should fix the end2end test failure.

**Review by AbdBarho — APPROVED** [review]
Thanks

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**